drivers/LVHDSR.py

```diff
@@ -892,7 +892,7 @@ def _handleInterruptedCoalesceLeaf(self):
         entries = self.journaler.getAll(cleanup.VDI.JRN_LEAF)
         if len(entries) > 0:
             util.SMlog("*** INTERRUPTED COALESCE-LEAF OP DETECTED ***")
-            cleanup.gc_force(self.session, self.uuid)
+            cleanup.start_gc_service(self.uuid, True)
```

Comment:
No, this really does not mean the same thing as `gc_force` just leave it as `gc_force`